packages/vendure-plugin-modify-customer-orders/test/tsconfig.e2e.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
```

Comment:
Are all the fields in `compilerOptions` necessary? They are similar to the parent/parent Tsconfig right? It would be nice if we dont need extra overrides, but I'm not sure if you have tested this.